leetcode-extension/background.js

```diff
@@ -15,9 +15,9 @@ query {
 
 // this function will retry for 3 times if any error occur while fetching the leetcode graphql
 let fetchLeetcodeData = async () => {
-  let retries = 3;
-  let success = false;
-  while (retries > 0 && !success) {
+  let numRetries = 3;
```

Comment:
Can we have this variable name to maxRetries ? As this api may generate response in any of 3 tries if some error occured.